code/style.css

```diff
@@ -1,58 +1,145 @@
-body {
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body,
+html {
+  height: 100%;
   margin: 0;
   font-family: "Courier New", Courier, monospace;
 }
 
+main {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  justify-content: center;
+  background-color: #07080b;
+}
+
 p {
   margin: 1px;
   font-weight: 300;
-  background-color: #fff;
+  background-color: transparent;
+}
+
+.hero-image {
+  height: 50%;
+  background-position: center;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-wrap: wrap;
+  object-fit: cover;
+}
```

Comment:
Great idea to have one large image as a background. It can be tricky with it causing the scrollbars, though. Perhaps try some of the tricks in the links below? 
https://css-tricks.com/perfect-full-page-background-image/  
https://www.w3schools.com/html/html_images_background.asp